doc/guides/modules.rst

```diff
@@ -10,14 +10,342 @@ system those are called *modules*. These modules must be integrated with the
 Zephyr build system, as described in more detail in other sections on
 this page.
 
-Zephyr depends on several categories of modules, including:
+To be classified as a candidate for being included in the default list of
+modules, an external project is required to have its own life-cycle outside
+the Zephyr Project, that is, reside in its own repository, and have its own
+contribution and maintainance workfow and release process. Zephyr modules
+should not contain code that is written exclusively for Zephyr.
+
+  .. note::
+
+     Code which is written for Zephyr and works exclusively with Zephyr
+     should not be submitted as a Zephyr module; instead, it should be
+     part of the main Zephyr tree.
+
+Modules to be included in the default manifest of the Zephyr project need to
+provide functionality or features endorsed and approved by the project technical
+steering committee and should comply with the
+:ref:`module licensing requirements<modules_licensing>` and
+:ref:`contribute guidelines<modules_contributing>`. They should also have a
+Zephyr developer that is commited to maintain the module code base.
+
+Zephyr depends on several categories of modules, including but not limited to:
 
 - Debugger integration
 - Silicon vendor Hardware Abstraction Layers (HALs)
 - Cryptography libraries
 - File Systems
 - Inter-Process Communication (IPC) libraries
 
+This page summarizes a list of policies and best practices which aim at
+better organizing the workflow in Zephyr modules.
+
+Module Repositories
```

Comment:
I think all this additional text actually belongs in the contributions guides: https://docs.zephyrproject.org/latest/contribute/index.html

and not in the `Modules (External project)` user guide.

The existing Module user guide is useful for a downstream developer who wants to include his own code/repository as a Zephyr module, but that information his now placed after a lot of text that is not relevant if you are creating you own module.

I would like to keep those two sections completely separated.

Feel free to drag the existing description into a `contribute/Modules` page.
https://docs.zephyrproject.org/latest/guides/modules.html#submitting-a-new-module

Reply:
Discussed in the meeting. 
I will keep the whole guide about module here, to make it easier for people to review the content, and after it is finalize, we will start the action of splitting the content in different areas in the project documentation.

Reply:
@ioannisg I know, I just placed a comment here for all of those looking in github, but not attending the meeting, would know :wink: